src/Pate/Verification/Widening.hs

```diff
@@ -231,7 +232,7 @@ widenPostcondition bundle preD postD0 =
                  -- https://github.com/GaloisInc/what4/issues/196
                  W4.push sp
                  W4.assume (W4.solverConn sp) precond
-            widenLoop sym bak initialGas eqCtx postD0 Nothing
+            widenLoop sym bak (Gas 100) eqCtx postD0 Nothing
```

Comment:
Can we just lift this to a named constant at the top-level? I presume we'll want to make it a parameter eventually, but making it clearer that this is a distinct value might be useful